portfolio-alex.html

```diff
@@ -0,0 +1,187 @@
+<!doctype html>
+
+<html lang="en">
+	
+	<head>
+		
+		<link rel="stylesheet" type="text/css" href="portfolio-alex.css"/>
+		
+		<link href="https://fonts.googleapis.com/css?family=Spartan&display=swap" rel="stylesheet"/>
+		
+		<meta name="viewport" content="width=device-width,initial-scales=1"/>
+		
+		<title>Alex Jennings Portfolio</title>
+		
+	</head>	
+	
+	<body>
+			
+		<header class="topheader">
+					
+				<div class="logo">
+						
+					<div class="logotext">
+					
+						<h1>ALEX JENNINGS</h1>
+						
+						<p>Full Stack Developer</p>
+							
+					</div>
+				
+				</div>
+				
+		 	 	 	<nav class="navlinks">
+					
+						<div class="navholder">
+	  
+	  						<strong>
```

Comment:
strong tags are designed to emphasis content, if you want these links bold, use the css and remove the strong tag